### PR TITLE
Add NumPy constants to ChainerX

### LINKED ISSUE #7368: VAE example does not work with ChainerX
Version: latest master(ac4e62d3d5c53069d1f4a224989396a28b19c3d8)
Running VAE example with a ChainerX device results in an error.

```
$ ./train_vae.py --device native:0
...
  File "/home/niboshi/w/repos/chainer/chainer/distributions/bernoulli.py", line 51, in backward
    nan = xp.array(xp.nan).astype(dlogit.dtype)
AttributeError: module 'chainerx' has no attribute 'nan'
```

### PR DESCRIPTION
**Pull request by niboshi**
Fixes #7368 

See also: https://docs.scipy.org/doc/numpy/reference/constants.html

### CODE REVIEW

File: chainerx/__init__.py [hunk 1]
```diff
@@ -15,6 +15,12 @@
 
 if _available:
     from numpy import dtype  # NOQA
+    from numpy import (  # NOQA
+        Inf, Infinity, NAN, NINF, NZERO, NaN, PINF, PZERO,
+        e, euler_gamma,
+        inf, infty, nan,
+        newaxis,
```
Comment (hvy) [inline chainerx/__init__.py]: `newaxis` is already defined in the python binding in `python/array_index.cc`. How about removing the binding code?

Comment (niboshi) [reply chainerx/__init__.py]: Thanks, fixed as suggested.

**Comment by hvy** [comment]
Jenkins, test this please.

**Comment by hvy** [comment]
LGTM, failure is unrelated.

**Pull request merged**